src/ko/index.html

```diff
@@ -302,12 +302,12 @@ <h1>피카츄 배구 - 슈퍼 AI 버전</h1>
 					</p>
 					<hr style="width: 100%;">
 					<p class="reverse-engineered-by">
-						<a href="https://gorisanson.github.io/pikachu-volleyball/ko/" target="_blank">웹 버전</a>을 바탕으로
-						<a href="https://gorisanson.github.io/" target="_blank">이규태</a>가 개발한 것입니다.
+						<a href="https://gorisanson.github.io/pikachu-volleyball/ko/" target="_blank">피카츄 배구
+						웹 버전</a>(만든 사람: <a href="https://gorisanson.github.io/" target="_blank">이규태</a>)에 기반합니다.
 					</p>
 					<hr style="width: 100%;">
 					<p class="reverse-engineered-by">
-						<a href="https://blog.duckll.tw/" target="_blank">DuckLL</a> 두 번째 수정 적용
+						슈퍼 AI 버전을 만든 사람: <a href="https://blog.duckll.tw/" target="_blank">DuckLL</a>
```

Comment:
The original Korean expression meant:
DuckLL applied second modification.

I feel like this expression was somewhat vague ("What is second modification?").
So I changed it to mean:
"Super AI version is made by DuckLL"